### PR TITLE
First conda package for tsam

### PR DESCRIPTION
**Pull request by julian-belina**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mattkeller304** [comment]
I accept to be maintainer

**Comment by julian-belina** [comment]
@conda-forge/staged-recipes please merge the pull request

File: recipes/tsam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set name = "tsam" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/refs/heads/master.tar.gz
```
Comment (ocefpaf) [inline recipes/tsam/meta.yaml:9]: We need a stable tag for this. We cannot package a dev release from `master`.

**Review by ocefpaf — CHANGES_REQUESTED** [review]
We need a tag/release for all the packages.

File: recipes/tsam/meta.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {% set name = "tsam" %}
-{% set version = "2.0.0" %}
+{% set version = "2.2.0" %}
```
Comment (ocefpaf) [inline recipes/tsam/meta.yaml:2]: The latest stable tag is https://github.com/FZJ-IEK3-VSA/tsam/releases/tag/v2.0.0
Can you ask upstream to release a 2.2.0? Or maybe base your recipe on the 2.0.0?

Comment (julian-belina) [reply recipes/tsam/meta.yaml]: Thank your very much for your review. There was a mistake in the version number of the commits. The code is based on version 2.0.0. I Incorporated your request.

File: recipes/tsam/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{% set name = "tsam" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/49bfc92a2c19c088efd5bdf0fb00e05c7d4383b0.tar.gz
```
Comment (ocefpaf) [inline recipes/tsam/meta.yaml:9]: Our infrastructure will use the version number to send updates here.

```suggestion
  url: https://github.com/FZJ-IEK3-VSA/tsam/archive/refs/tags/v{{ version }}.tar.gz
```

File: recipes/tsam/meta.yaml (hunk 1 above)
Comment (julian-belina) [reply recipes/tsam/meta.yaml]: A stable tag has been added in the lattest commit

**Pull request merged**